api/api.js

```diff
@@ -60,8 +44,8 @@ module.exports = {
         });
     },
 
-    getById: function(req, res) {
-        return EntryModel.findById(req.params.id, function(err, entry) {
+    getById: function(req, res, model) {
+        return model.findById(req.params.id, function(err, entry) {
```

Comment:
Careful - doesn't the .findById() function use MongoDB's automatically added ID? As opposed to PlayerID?